observation_portal/requestgroups/serializers.py

```diff
@@ -749,7 +749,8 @@ def validate(self, data):
                 raise serializers.ValidationError(
                     (
                         'According to the constraints of the request, the target is visible for a maximum of {0:.2f} '
-                        'hours within the time window. This is less than the duration of your request {1:.2f} hours. '
+                        'hours within the future time window. '
```

Comment:
"the target will only be visible for a maximum of {0:.2f} hours within the time window"